### ISSUE TITLE
How change the poster of a show?

### ISSUE THREAD

**Issue by pucelano7** [pymedusa/Medusa#3585]
I use TVDb2 for get the metadata of the shows...but with some TV shows the poster that Medusa shows is not correct. Can I change it manually or force the change?

For example: with Mr.Robot i have the next poster (the rest of the metadata is correct)

![Uploading 0A2E44D9-FF07-4821-B65B-0C3B0186C54C.png…]()

**Comment by p0psicles** [comment]
You can run a `force update` for the show. Have you tried that?

**Comment by pucelano7** [comment]
Yes, I did it.
Also I removed the show from Medusa and added it again...but the error continues

**Comment by p0psicles** [comment]
Okay with error you mean the wrong poster?
Also can you show your poster? When adding an image to github. You have to wait a sec before saving your comment. In your initial comment, you where to fast.

**Comment by pucelano7** [comment]
Yes, the error is the wrong poster
Sorry, I upload it again :-)
![7f615fe5-eab5-413a-a389-064bb3125eb0](https://user-images.githubusercontent.com/24883267/34716174-5ca79fe4-f52f-11e7-872c-688bf919a0ff.png)

**Comment by eckhardt83** [comment]
I have same problem ..

**Comment by p0psicles** [comment]
Tried with multiple branches. But can't reproduce your issue. Have you maybe ever used a different indexer to add this show? Maybe tmdb or tvmaze?

Else please enable debugging and restart.
Then go into mr. robot, and run a force update.

Post the debug log while running the update.

**Comment by eckhardt83** [comment]
in my case it it Arrow where the problem is ( see link)
[link](https://imgur.com/a/OxqNK)

debug log:
[debug log link](https://pastebin.com/WADk0T94)

**Comment by p0psicles** [comment]
I'm only seeing the show refresh. Please do the following. Go into your show. Arrow in this case. And start a force update. Post the logs of that.

**Comment by eckhardt83** [comment]
I have tryet again I am using the "Force Full Update"

log
[](https://pastebin.com/Xawt7r8Q)

**Comment by p0psicles** [comment]
I'm only seeing the SHOW-REFRESH part of the force full update. Before that you should have some logs that look like this:
```
2018-01-11 17:21:39 DEBUG    SHOWQUEUE-UPDATE :: [ef4b3e6] 308030: Retrieving show info from TVDBv2
2018-01-11 17:21:39 DEBUG    SHOWQUEUE-UPDATE :: [ef4b3e6] 308030: Using language from show settings: en
2018-01-11 17:21:39 DEBUG    SHOWQUEUE-UPDATE :: [ef4b3e6] 308030: Loading show info from TVDBv2
2018-01-11 17:21:39 DEBUG    SHOWQUEUE-UPDATE :: [ef4b3e6] 308030: Using language from show settings: en
2018-01-11 17:21:39 DEBUG    SHOWQUEUE-UPDATE :: [ef4b3e6] Configured language en override show language of en
2018-01-11 17:21:39 DEBUG    SHOWQUEUE-UPDATE :: [ef4b3e6] Getting all series data for 308030
2018-01-11 17:21:39 DEBUG    SHOWQUEUE-UPDATE :: [ef4b3e6] Getting all show data for 308030
```

**Comment by Teryon** [comment]
Same Problem here bad poster and fanart for arrow (TVDBv2), no pictures at all for flash (TVDBv2), I've checked on TVDB and there is all art needed for this show.
Here is debug for arrow :
https://pastebin.com/D31pDeMS
Here is debug for flash :
https://pastebin.com/bnewRuBh

It does not happen only on TVDBv2, it would be good to be able to change art choosing from the different choice of the indexer.

**Comment by medariox** [comment]
There are no French posters for the show, change it to English.

**Comment by Teryon** [comment]
When I go to
https://www.thetvdb.com/?tab=series&id=279121&lid=17 (Flash)
https://www.thetvdb.com/?tab=series&id=257655&lid=17 (Arrow)
There is posters !

**Comment by medariox** [comment]
Yeah, but no *French* posters.

**Comment by p0psicles** [comment]
Yes, they are all english. Click on them.
![image](https://user-images.githubusercontent.com/1331394/36733046-b5a53dc4-1bcf-11e8-960f-e593e4741c59.png)

**Comment by Teryon** [comment]
Ok I've seen what you mean and I've added one for french :D

**Comment by Teryon** [comment]
Is it the same for fanart ? arrow is displaying one when flash don't, and arrow has no fanart for french ?

PS : a full rescan for arrow & TVDB has fixed the problem !, it only happen when you add the shows it download bad arts.

**Comment by Teryon** [comment]
Is it the same for fanart ? arrow is displaying one when flash don't, and arrow has no fanart for french ?

PS : a full rescan for arrow & TVDB has fixed the problem !, it only happen when you add the shows it download bad arts.

Trying to add "Go on" show, and it is loading a poster that doesn't correspond at all with the show and doesn't appear on the show page on tvdb, even a force full update doesn't change anything !

**Comment by Teryon** [comment]
OK need to have a rating, to work ...

**Comment by Teryon** [comment]
Whatever I do a show gets the most rated artwork, whether it is spanish (like young sheldon on tmdb) or some art I don't even kow when adding westworld on tvdb, coming from tmdb : http://image.tmdb.org/t/p/original/pYL3YYNMwGz18dylTxRM8Ffai4e.jpg

Or there is something I'm missing :/

**Comment by medariox** [comment]
We can't help you without debug logs.

**Comment by Teryon** [comment]
full scan update on young sheldon on TMDB
https://pastebin.com/5f77rWbC

**Comment by medariox** [comment]
What's the issue with that show?

**Comment by Teryon** [comment]
Westworld loading some artwork from TMDB that has nothing to do with westworld
Young sheldon, is getting spanish poster while there is a french poster

Wouldn't it be better to get english art when there isn't any french one, instead of getting something from TMDB, or not getting anything at all ?

**Comment by Teryon** [comment]
nobody ?

**Comment by p0psicles** [comment]
Pls have patience with this. It's not a very high prio issue. But I'll look at it eventually.

**Comment by Teryon** [comment]
Ok no problems, I tought it'd been missed.
Thanks for all.

**Comment by EraYaN** [comment]
Essentially when the main language poster/fanart/image is missing, there is no proper fallback.

When there is a poster but no fanart in the language it DOES work correctly, and does not download a wrong fanart, it just leaves it blank.

Look at this show for example: [Life Bar](https://www.thetvdb.com/series/life-bar) The only poster is Korean language (and that is correct seeing the text on the poster is in fact in Korean).

I would argue that even when I pick "English" as the metadata language, it should not download [this Barbie poster](https://www.thetvdb.com/download/942869). It should either accept that there is no poster OR better yet, fallback to the next language (for example the source language of the show would be a good first try, in this case Korean) which would yield [this file](https://www.thetvdb.com/download/1179007).

**Comment by FransAlmere** [comment]
I know this topic is closed but I just found a work around. Had the same problem with the Serie Das Boot from Amazon, got a ridilicous poster from some children serie. But found out that on my synology the file was stored in the path /volume1/@appstore/sickbeard-custom/var/cache/images/tvdb/thumbnails. Than I looked it up (took some time) but found it with ****.jpg. Deleted that file looked up the original thumb file from Amazon and replaced it with this one. Be sure you use the same number as the original file in my case for example 352882.jpg. I do not know wat happens if you simple delete the file and than do a forced update.

**Comment by adocampo** [comment]
Thank you @FransAlmere I have more than 100 shows and 15 or so have the wrong poster and background. I can do that for the shows are appearing wrong.
To the devs, I know this is just a cosmetic problem and not a _real_ issue, but it would be great to let users to upload their custom images or specify the right URL. :)

**Comment by p0psicles** [comment]
You can also enable the Metadata options for banner, poster, fanart. Then the images are stored in the show folder. You can overwrite those images

**Comment by adocampo** [comment]
Mmmm, where can I enable those options, I'd navigated through the global settings and per tvshow, but I cannot see anywhere anything related to the metadata.

**Comment by p0psicles** [comment]
Post-processing - Metadata. Use kodi v12

**Comment by FransAlmere** [comment]
![Medusa Postprocess Kodi 12](https://user-images.githubusercontent.com/13037978/56715399-df604000-6737-11e9-8516-26781a8ba07f.jpg)
As you can see I allready had the options as mentioned by p0psicles in Medusa PostProcessing enabled but that still did not help. Perhaps a problem in combination with Synology ? I do not now.

But for now me the problem is solved.

**Comment by p0psicles** [comment]
@fransalmere did it create the poster in your show folder?

**Comment by FransAlmere** [comment]
Yes, it did create a poster but the wrong one and the only I could that problem was as described before !

**Comment by adocampo** [comment]
I finally changed the language of metadata to english on those failing tvshows. Now it works. I'll take a look this evening to see if post-processing settings can do something.
